ee/tabby-webserver/src/schema.rs

```diff
@@ -29,6 +36,35 @@ impl Mutation {
         let token = ctx.reset_registration_token().await?;
         Ok(token)
     }
+
+    async fn user_register(
+        ctx: &ServerContext,
+        input: RegisterInput,
```

Comment:
do not consolidate a struct to hold input parameters, this will generate different api.

Django's
```
mutation {
  register(
    email:"skywalker@email.com",
    username:"skywalker",
    password1: "qlr4nq3f3",
    password2:"qlr4nq3f3"
  ) {
    success,
    errors,
    token,
    refreshToken
  }
}
```

Your implementation
```
mutation {
  register(input: {
    ...
  })
}
```